api/logs/serializers.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import serializers as ser
+
+from website.models import NodeLog
+
+from api.base.serializers import JSONAPISerializer, LinksField, Link
+
+class LogSerializer(JSONAPISerializer):
+
+    filterable_fields = frozenset(['action'])
+
+    id = ser.CharField(read_only=True, source='_id')
+
+    date = ser.DateTimeField(read_only=True)
+    action = ser.CharField(help_text='One of: {0}'.format(', '.join(NodeLog.actions())))
+    params = ser.DictField()
+
+    nodes_logged = ser.SerializerMethodField(help_text='A list of node primary keys for nodes associated with this log')
+    user = ser.SerializerMethodField(help_text='The user associated with this log')
+
+    def get_user(self, log):
+        return log._render_log_contributor(log.user._id)
+
+    def get_nodes_logged(self, log):
+        return [n._id for n in log.node__logged]
+
+    links = ser.SerializerMethodField()
+
+    def get_links(self, log):
+        links = {
+            'nodes': {
```

Comment:
@brianjgeiger what is the convention for 'related' links that need to refer to a list of resources?

For example, a log may be attached to more than one Node. It would be nice to have links for each related node. Is the list implemented here OK?